CONTRIBUTING.md

```diff
@@ -4,6 +4,10 @@
 
 In this project, we are heavily utilising GitHub features to document and signal any progress in the website development.
 
+### Finding or creating issues
+
+Issues is one of many features in Github. Most contributions start from defining issues. Anyone can start to open an issue for discussion. You can head to [this link](https://guides.github.com/features/issues/) for deep understanding about Issues. Specifically, you can start finding several Issues in [here](https://github.com/kawalcovid19/wargabantuwarga.com/issues). There are only 2 categories in Issues section. They are open and closed Issue. Open Issues means it still needs more attention and need to be resolved. Closed Issues means it has been completed or no need further action.
```

Comment:
```suggestion
Most contributions start from defining issues. Anyone can open an issue for discussion. You can head to [this link](https://guides.github.com/features/issues/) for deep understanding about Issues. Specifically, you can start finding several Issues in [here](https://github.com/kawalcovid19/wargabantuwarga.com/issues). There are only 2 categories in Issues section, Open and Closed. Open Issues means it still needs more attention and need to be resolved. Closed Issues means it has been completed or no need further action.
```

Reply:
What about "Issues" as the name of feature in github? Should it be plural or singular?